sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1664,6 +1664,30 @@ def boundary_conditions(self):
         """
         return self._boundary_conditions
 
+
+    def polar_moment(self):
+        """
+        This function calculates the Polar moment of Inertia of the beam about the x axis.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.continuum_mechanics.beam import Beam3D
+        >>> from sympy import symbols
+        >>> l, E, G, I, A = symbols('l, E, G, I, A')
+        >>> b = Beam3D(l, E, G, I, A)
+        >>> b.polar_moment()
+        2*I
+        >>> I1 = [9, 15]
+        >>> b = Beam3D(l, E, G, I1, A)
+        >>> b.polar_moment()
+        24
+        """
+        if isinstance(self.second_moment, int) or isinstance(self.second_moment, Symbol):
```

Comment:
This is not a good check. You should check if it iterable or if it has a length of 2 to know if there is a unique Ix and Iy.